stetho/src/main/java/com/facebook/stetho/inspector/network/NetworkPeerManager.java

```diff
@@ -50,9 +51,19 @@ public AsyncPrettyPrinterRegistry getAsyncPrettyPrinterRegistry() {
     return mAsyncPrettyPrinterRegistry;
   }
 
+  public void setPrettyPrinterInitializer(AsyncPrettyPrinterInitializer initializer) {
+    mPrettyPrinterInitializer = Util.throwIfNull(initializer);
```

Comment:
yeah basically make the set call non idempotent. This way you can do things like only fire the initializer once. Unless im mistaken there is no real need for the client process to change what pretty printers are registered during a single process invocation?